evap/staff/views.py

```diff
@@ -903,19 +903,42 @@ def semester_questionnaire_assign(request, semester_id):
     course_types = CourseType.objects.filter(courses__evaluations__in=evaluations)
     form = QuestionnairesAssignForm(request.POST or None, course_types=course_types)
 
+    general_fields = [field for field in form if field.name.startswith("general-")]
+    contributor_fields = [field for field in form if field.name.startswith("contributor-")]
+    contributor_fields.append(form["all-contributors"])
```

Comment:
These can be moved down, because they are not needed in the `form.is_valid()` case